Project.toml

```diff
@@ -25,7 +25,7 @@ FastGaussQuadrature = "0.5, 1"
 FillArrays = "0.13, 1"
 LRUCache = "1"
 LinearAlgebra = "1"
-QuantumInterface = "0.3.3"
+QuantumInterface = "0.3.9"
```

Comment:
Any reason for the bump from 3 to 9?

Reply:
Right! Sorry for the noise, I left this comment before looking at the PR over at QuantumInterface